manager/controllers/packages.go

```diff
@@ -43,11 +43,19 @@ func packagesQuery(acc string) *gorm.DB {
 
 // @Summary Show me all installed packages across my systems
 // @Description Show me all installed packages across my systems
-// @ID packages
+// @ID listPackages
 // @Security RhIdentity
 // @Accept   json
 // @Produce  json
-// @Success 200 {object} PackageSystemsResponse
+// @Param    limit          query   int     false   "Limit for paging, set -1 to return all"
```

Comment:
This is different thing, right? Wouldn't be better to put it to sepparate pr?

Reply:
Removed